### PR TITLE
Creating Calculator components

### PR DESCRIPTION
**Pull request by cosywasswa**
I performed the following tasks.

- [ ] Created a new branch called calc-components.
- [ ] Removed the original boilerplate elements(text, images,styles)
- [ ] Created a directory called components with calculator.js file.
- [ ] Created Calculator code with React and imported it to the App.js
- [ ] Created styles for the calculator.
- [ ] Fixed linter errors.

### CODE REVIEW

**Review by GzimAsani — CHANGES_REQUESTED** [review]
Hi @cosywasswa , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that does not have something under the future features on the readme file. Just adding Calculator is not explanatory enough and moreover is what you are doing now, you need to add some extra features that you might want to do it latter. 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: src/components/calculator.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+              <td><button type="button" className="others">.</button></td>
+              <td><button type="button" className="operators">=</button></td>
+            </tr>
+          </tbody>
+        </table>
+      </div>
+
+    );
+  }
+}
```
Comment (levy002) [inline src/components/calculator.js]: - [ ] Kindly make sure to make this class component to `functional component` in order to meet the project requirement.
- [ ] Kindly make sure this `calculator.js` component has at least 1 child component in order to match the project requirement.

File: src/App.js [hunk 2]
```diff
@@ -1,29 +1,17 @@
...
+  constructor(props) {
+    super(props);
+    this.state = {};
+  }
+
+  render() {
+    return (
+      <Calculator />
+    );
+  }
```
Comment (levy002) [inline src/App.js]: - [OPTIONAL] Kindly make sure to change this Class component to `functional component`.

**Review by levy002 — CHANGES_REQUESTED** [review]
## Hello @cosywasswa  , :wave:
### I'm @levy002, your PR reviewer

Good job so far :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- No linter error :heavy_check_mark: 

## Status: REQUIRED CHANGES ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @levy002 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/calculator.js [hunk 3]
```diff
@@ -1,58 +1,52 @@
...
-              <td colSpan="2"><button type="button" className="others">0</button></td>
-              <td><button type="button" className="others">.</button></td>
-              <td><button type="button" className="operators">=</button></td>
-            </tr>
-          </tbody>
-        </table>
-      </div>
+function Display() {
+  return <input type="text" className="result" placeholder="0" />;
+}
```
Comment (levy002) [inline src/components/calculator.js]: - [ ] Well done on creating this child component, however, it is always advisable to create the unique file and import it where you want to use it. it's not a good practice to have 2 components in one file.

  **_Kindly go ahead and create a new file for the `Display` component and export it and after all, import it in this file in order to use it._**

**Review by levy002 — CHANGES_REQUESTED** [review]
## Hello @cosywasswa   , :wave:
### I'm Levy, your PR reviewer

Good job so far :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!



## Status: REQUIRED CHANGES ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @levy002 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Olamarx — APPROVED** [review]
Hi @cosywasswa,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- Required changes fixed ✔️
- Linters passed ✔️
- Commit messages meaningful ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**